dev/io.openliberty.http.monitor_fat/test-applications/JsfApp/resources/normal.html

```diff
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='UTF-8' ?>
```

Comment:
missing copyright

Reply:
I don't think we usually copyright HTML files?